deluge/ui/gtkui/statusbar.py

```diff
@@ -326,6 +331,12 @@ def _on_max_upload_speed(self, max_upload_speed):
         self.max_upload_speed = max_upload_speed
         self.update_upload_label()
 
+    def _on_get_external_ip(self, external_ip):
+        if external_ip:
+            self.external_ip_item.set_markup(_("<b>IP</b> %s") % external_ip)
+        else:
+            self.external_ip_item.set_text(_("<b>IP</b> n/a"))
```

Comment:
This should be set_markup() if you don't change the function to remove this line.